src/tiledb/cloud/files/utils.py

```diff
@@ -164,3 +170,78 @@ def export_file(
         )
     except GenApiException as exc:
         raise tiledb_cloud_error.check_exc(exc) from None
+
+
+def upload_file(
```

Comment:
Like the `export_file` can we also tie this into the current `create_file` (I like the name upload_file btw) but wanting to see if we can switch automatically on "local upload" vs "load from s3"?

Reply:
This is a good idea for a followup.

Reply:
Let's get this merged then, and can we schedule the followup soon, @antalakas ?